pages/Documents/RichMessaging/StructuredContent/AppleBusinessChatTemplates/rich-links-template.md

```diff
@@ -4,7 +4,7 @@ Keywords:
 sitesection: Documents
 categoryname: "Rich Messaging"
 documentname: Link Previews
-permalink: link-previews-apple-business-chat-template.html
+permalink: link-previews-apple-messages-for-business-template.html
```

Comment:
Please add a redirect so that the old path keeps working.

Reply:
reverted -- will address page names and redirects separately